Source/UI/NewUI/CommonUI/View/Component/InputView.swift

```diff
@@ -0,0 +1,195 @@
+import UIKit
+
+class InputView: UIView {
+
+    // MARK: - Properties
+
+    weak var delegate: TextFieldViewDelegate?
+    private(set) var style: CellTextFieldStyle?
+    private(set) lazy var textFieldContainerBottomAnchor = textFieldContainer?.bottomAnchor.constraint(equalTo: bottomAnchor, constant: 0)
```

Comment:
Feels like this could easily break if setup is moved out of init, so could we just have an optional property that gets assigned as part of the relevant setup method?

Reply:
most of code can be easily break if it's move from the right place